sale_order_tag/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "author": "Patrick Wilson, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
     "category": "Sale",
-    "version": "14.0.1.0.1",
+    "version": "14.0.1.0.2",
```

Comment:
@alexeirivera87 FYI, you don't need to change this unless you provide migration scripts (it's done automatically by bot at merge).